srcpkgs/a4/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'a4'
+pkgname=a4
+version=0.2.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="libtool pkg-config"
+makedepends="libvterm-devel libtickit-devel"
+depends="libvterm>=0.3.1 libtickit>=0.4.3"
```

Comment:
Is it really required to manually define minimum versions here?

Reply:
Hi Duncan, 

many thanks for your reply, regarding this i'm not sure if this is necessary inside the template, but it is a requirements of the original software: https://github.com/rpmohn/a4 (inside the readme I found  this requirements). 

before submitting this new package I already update the void templates for this required packages with this PRs: 
(https://github.com/void-linux/void-packages/pull/43532)
(https://github.com/void-linux/void-packages/pull/43658)

So i'm not sure if the line you mention now in this template is needed or not, please help me to understand

Many thanks

Reply:
Defining `depends` manually will overwrite the default minimum version based on the shared library version. 

```
void-packages% cat common/shlibs|grep libvterm
libvterm.so.0 libvterm-0.3_1
void-packages% cat common/shlibs|grep libtickit
libtickit.so.3 libtickit-0.4.1_1
```

Just for simplicity and for future updates I prefer not to manually overwrite this.